libs/source_post_processors/objects_filtering.py

```diff
@@ -1,13 +1,23 @@
+import os
+import cv2 as cv
 import numpy as np
 
+from pathlib import Path
+
+from ..utils.loggers import get_source_log_directory
+
 
 class ObjectsFilteringPostProcessor:
 
     def __init__(self, config, source: str, post_processor: str):
         self.config = config
+        self.source = source
         self.overlap_threshold = float(
             self.config.get_section_dict(post_processor)["NMSThreshold"]
         )
+        camera_id = config.get_section_dict(source)["Id"]
+        roi_file_path = ObjectsFilteringPostProcessor.get_roi_file_path(camera_id, config)
```

Comment:
I there any reason why you are using `ObjectsFilteringPostProcessor` instead of `self`.

Reply:
Self isn't only for instances?